jwst/assign_wcs/nirspec.py

```diff
@@ -805,7 +824,8 @@ def gwa_to_ifuslit(slits, input_model, disperser, reference_files):
     return Gwa2Slit(slits, slit_models)
 
 
-def gwa_to_slit(open_slits, input_model, disperser, reference_files):
+def gwa_to_slit(open_slits, input_model, disperser,
+                reference_files, slit_y_range=[-.55, .55]):
```

Comment:
It doesn't look as if you passed `slit_y_range` to this function where it is called.